src/app/seamly2d/core/vtooloptionspropertybrowser.cpp

```diff
@@ -746,6 +746,21 @@ void VToolOptionsPropertyBrowser::addPropertyLineColor(Tool *tool, const QString
     addProperty(lineColorProperty, id);
 }
 
+//---------------------------------------------------------------------------------------------------------------------
+template<class Tool>
```

Comment:
Please add comments per this guide: 
https://developer.lsst.io/cpp/api-docs.html#cpp-doxygen-class-structure